src/pages/en/guides/deploy.md

```diff
@@ -161,21 +161,13 @@ jobs:
 
 ## GitLab Pages
```

Comment:
Can I please get someone to check this example, @delucis @aFuzzyBear? Step 2 says to Set `dist` but the proposed example doesn't include `dist` in the new config.  :)